rest/models/model_rows_response.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type RowsResponse struct {
+	Success bool `json:"success,omitempty"`
+
+	RowsModified int32 `json:"rowsModified,omitempty"`
```

Comment:
Unlike SQL, INSERT and UPDATE supports updating 1 row at a time, there are other features (like static cols) that makes `rowsModified` even more confusing for the user. We should remove this property.

Reply:
Oh, I didn't know that was a thing. I thought it always updated every row matching the condition.